project/dexter-server/public/AdminSE/adminSECtrl.js

```diff
@@ -1,101 +1,200 @@
 "use strict";
-adminSEApp.controller('adminSECtrl', function($scope, $http, $location ,$routeParams, $log) {
+adminSEApp.controller('adminSECtrl', function ($scope, $http, $location, $routeParams, $log) {
     const NONE_MODULE_PATH = 'undefined';
 
-    $('select').on('select2:select', function (event) {
+    $('.did-select').on('select2:select', function (event) {
+        addDidList(event);
+        changeSelectedDidCount();
+    });
+
+    $('.module-select').on('select2:select', function (event) {
         addModulePathList(event);
         changeSelectedModuleCount();
     });
 
-    $('select').on('select2:unselect', function (event) {
+    $('.did-select').on('select2:unselect', function (event) {
+        removeDidList(event);
+        changeSelectedDidCount();
+    });
+
+    $('.module-select').on('select2:unselect', function (event) {
         removeModulePathList(event);
         changeSelectedModuleCount();
     });
 
-    function changeSelectedModuleCount(){
-        var count = $scope.selectedModulePathList.length;
-        $("#removeModuleCount").html(parseInt(count));
-        if(count > 0) {
+    function changeSelectedModuleCount() {
+        const count = parseInt($scope.selectedModulePathList.length);
+        $("#removeModuleCount").html(count);
+        if (count > 0) {
             $("#isModuleSelected").removeClass("fa fa-folder").addClass("fa fa-folder-open");
-        }else{
+        } else {
             $("#isModuleSelected").removeClass("fa fa-folder-open").addClass("fa fa-folder");
         }
     }
 
-    function addModulePathList(event){
-        if (!event) { return; }
+    function changeSelectedDidCount() {
+        const count = parseInt($scope.selectedDidList.length);
+        $("#removeDidCount").html(count);
+        if (count > 0) {
+            $("#isDidSelected").removeClass("fa fa-minus-square").addClass("fa fa-bug");
+        } else {
+            $("#isDidSelected").removeClass("fa fa-bug").addClass("fa fa-minus-square");
+        }
+    }
+
+    function addModulePathList(event) {
+        if (!event) {
+            return;
+        }
         JSON.stringify(event.params, function (key, value) {
             $scope.selectedModulePathList.push(value.data.id);
         });
     }
 
-    function removeModulePathList(event){
-        if (!event) { return; }
+    function removeModulePathList(event) {
+        if (!event) {
+            return;
+        }
 
         JSON.stringify(event.params, function (key, value) {
             $scope.selectedModulePathList.pop(value.data.id);
         });
     }
 
-    var init = function(){
-        isShownLoadingImage();
+    function addDidList(event) {
+        if (!event) {
+            return;
+        }
+
+        JSON.stringify(event.params, function (key, value) {
```

Comment:
@Minjung-Baek I think JSON.stringify() is not suitable for just looping.
Is not forEach() better ?